processor/src/tests/mod.rs

```
@@ -93,16 +103,27 @@ async fn test_send<C: Coin + Clone>(coin: C, fee: C::Fee) {
 
     let latest = coin.get_latest_block_number().await.unwrap();
     wallet.acknowledge_block(1, latest - (C::CONFIRMATIONS - 1));
-    let signable = wallet
-      .prepare_sends(1, vec![(wallet.address(), 10000000000)], fee)
+    let mut signable = wallet
+      .prepare_sends(1, vec![(wallet.address(), 4999900000)], fee)
       .await
-      .unwrap()
-      .1
-      .swap_remove(0);
-    futures.push(wallet.attempt_send(network, signable));
+      .unwrap();
+    if signable.1.len() <= 0 {
+      break;
```

This shouldn't break here. That invalidates this test by silently returning if it fails.